pkg/operator/k8sutil/network.go

```diff
@@ -119,3 +138,17 @@ func ApplyMultus(net rookv1.NetworkSpec, objectMeta *metav1.ObjectMeta) error {
 
 	return nil
 }
+
+// GetNetworkAttachmentConfig returns the NetworkAttachmentDefinitions configuration
+func GetNetworkAttachmentConfig(n netapi.NetworkAttachmentDefinition) (NetworkAttachmentConfig, error) {
+	netConfigJSON := n.Spec.Config
+	var netConfig NetworkAttachmentConfig
+
+	err := json.Unmarshal([]byte(netConfigJSON), &netConfig)
+	if err != nil {
+		return netConfig, fmt.Errorf("failed to unmarshal netconfig json: %s. %v", netConfigJSON, err)
+	}
+
+	logger.Debugf("NetworkAttachmentDefinition config is %+v", netConfig)
```

Comment:
This seems too verbose to dump the whole struct even in debug mode

Reply:
It won't be the only place where we dump a whole struct but fine it's not a reason to carry on.